Dockerfile.windows

```
@@ -259,27 +259,36 @@ RUN C:\BuildTools\VC\Auxiliary\Build\vcvars64.bat && \
 
 # Compiled DLL from build stage
-COPY --from=build C:\\build\\bin\\HoundTTS.dll  C:\\dist\\base\\Mods\\Services\\HoundTTS\\bin\\HoundTTS.dll
+COPY --from=build C:\build\bin\HoundTTS.dll  C:\dist\base\Mods\Services\HoundTTS\bin\HoundTTS.dll
 
 # Lua scripts + config from build context
-COPY dcs/Mods/Services/HoundTTS/Scripts/             C:\\dist\\base\\Mods\\Services\\HoundTTS\\Scripts\\
-COPY dcs/Mods/Services/HoundTTS/examples/            C:\\dist\\base\\Mods\\Services\\HoundTTS\\examples\\
-COPY dcs/Scripts/Hooks/HoundTTS-hook.lua              C:\\dist\\base\\Scripts\\Hooks\\HoundTTS-hook.lua
-COPY dcs/Config/HoundTTS-config.lua.example           C:\\dist\\base\\Config\\HoundTTS.lua.example
-COPY dcs/Config/HoundTTS-credentials.ini.example      C:\\dist\\base\\Config\\HoundTTS-credentials.ini.example
+COPY dcs/Mods/Services/HoundTTS/Scripts/             C:\dist\base\Mods\Services\HoundTTS\Scripts\
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)